src/js/player.js

```diff
@@ -49,7 +49,12 @@ vjs.Player = vjs.Component.extend({
     this.language_ = options['language'] || vjs.options['language'];
 
     // Update Supported Languages
-    this.languages_ = options['languages'] || vjs.options['languages'];
+    var languages = vjs.obj.copy(options['languages'] || vjs.options['languages']);
+    vjs.options['languages'] = {};
```

Comment:
Wrong solution to the problem. Or solution to the wrong problem. I need to rework that.

I've now realised that although `this.languages_` is either `options['languages']` or `vjs.options['languages'];`, `addLanguage()` always adds to `vjs.options['languages'];`. So a plugin which adds a language might not be able to add it to the player using it, but would add it to others on the page: http://jsbin.com/nomune/1/edit?html,console